swapi/src/app/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+  <form #loginForm="ngForm" (ngSubmit)="saveDetails()">
+    <mat-form-field>
+      <mat-label>Email:</mat-label>
+      <input
+        #email="ngModel"
+        matInput
+        type="text"
+        inputmode="email"
```

Comment:
not supported in safari

Reply:
works on iOS 16.1.1 
![IMG_0923](https://user-images.githubusercontent.com/48721238/225882642-1489211d-ad16-45ac-8a0d-f20a1f4474c4.PNG)

Reply:
not ios , but macos , check mdn ar caniuse